commons/addons/postgresql.md

```diff
@@ -8,6 +8,10 @@ position: 6
 PostgreSQL is an object-relational database management system (ORDBMS) with an emphasis on extensibility
 and on standards-compliance.
 
+## PostgreSQL version
+
+The version currently installed by the add-on is PostgreSQL 9.2.8
```

Comment:
The shared plans are using Postgresql 9.2.8
Newly created dedicated databases are currently using Postgresql 9.3.4